integration-tests/infinispan-client/pom.xml

```diff
@@ -128,17 +171,6 @@
             </resource>
         </resources>
         <plugins>
-            <plugin>
-                <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
Note: This is already handled in IT parent. With this change, the native IT ran just fine in my fork.